src/components/main/MenuBar.js

```diff
@@ -0,0 +1,223 @@
+import React from 'react'
+
+import {
+  StyleSheet,
+  Dimensions,
+  View,
+  Text,
+  TouchableOpacity,
+  Image,
+} from 'react-native';
+
+import COLORS from '../../assets/Colors.js'
+import { Row } from 'native-base';
+import { colors } from 'react-native-elements';
+import i18n from 'i18n-js'
+import { NavigationEvents } from 'react-navigation'
+import Auth from '@aws-amplify/auth'
+
+
+export default class MenuBar extends React.Component {
+    constructor(props) {
+        super(props);
+        
+        this.state = {
+          color: '',
+          isAuthenticated: false,
+          error: false,
+        }
+      
+       this.voltar = require('../../assets/icons/ic_back.png')
```

Comment:
please change these variables to english. i.e. back, home, profile, reflorestaLogo,institutoTerraLogo